server/libs/domain/src/auth/auth.service.ts

```diff
@@ -112,49 +115,22 @@ export class AuthService {
     }
   }
 
-  async validateSocket(client: Socket): Promise<UserResponseDto | null> {
-    try {
-      const headers = client.handshake.headers;
-      const accessToken =
-        this.extractJwtFromCookie(cookieParser.parse(headers.cookie || '')) || this.extractJwtFromHeader(headers);
-
-      if (accessToken) {
-        const payload = await this.cryptoRepository.verifyJwtAsync<JwtPayloadDto>(accessToken, { secret: jwtSecret });
-        if (payload?.userId && payload?.email) {
-          const user = await this.userCore.get(payload.userId);
-          if (user) {
-            return user;
-          }
-        }
-      }
-    } catch (e) {
-      return null;
+  public async validate(headers: IncomingHttpHeaders): Promise<UserResponseDto> {
```

Comment:
This is still returning `UserResponseDto` instead of `AuthUserDto`. Should we explicitly convert to `AuthUserDto`?